nautobot/ui/src/components/SidebarNav.js

```diff
@@ -29,6 +31,21 @@ export default function SidebarNav() {
         CurrentContextIcon = appContextIcons[currentContext];
     }
 
+    const isDisabledLinkProps = (route) => {
+        if (!isEnabledRoute(route)) {
+            return {
+                color: "gray.500",
+                cursor: "not-allowed",
+                _hover: {
+                    color: "gray.500",
+                },
+                // Prevent link click event
+                onClick: (e) => e.preventDefault(),
```

Comment:
This did not work on my end. The suggested change however worked for me. Can you try it out?
```suggestion
                "pointer-events": "none",
```

Reply:
😭

Reply:
Are you on `8080` or `3000`

Reply:
I tried 8080 and it didn't work. For 3000, I assumed that no links are disabled right.

Reply:
NVM it worked, on 8080 it took a while to rebuild the UI sorry :(